api/proto/oap/collector/receiver/opentelemetry/opentelemetry.proto

```diff
@@ -23,7 +22,7 @@ message PostMetricsServiceRequest {
   // element. Intermediary nodes (such as OpenTelemetry Collector) that receive
   // data from multiple origins typically batch the data before forwarding further and
   // in that case this array will contain multiple elements.
-  repeated erda.oap.opentelemetry.ResourceMetrics resource_metrics = 1;
+  repeated .opentelemetry.proto.metrics.v1.ResourceMetrics resource_metrics = 1;
```

Comment:
strange prefix `.`

Reply:
The current package name is "opentelemetry", and the outermost package with the same name cannot be directly referenced.